packages/types/src/lib/interfaces.ts

```diff
@@ -496,7 +514,19 @@ export interface SignConditionECDSA {
  *
  */
 export interface ExecuteJsResponse {
-  signatures: any;
+  success?: boolean;
+  signatures:
+    | {
+        sig: {
+          r: string;
+          s: string;
+          recid: number;
+          signature: string; // 0x...
+          publicKey: string; // pkp public key
+          dataSigned: string;
+        };
+      }
+    | any;
```

Comment:
do we want to union this type with `any`? I think we want a rigid definition of the signature shares.

Reply:
I think I did it in another PR